test/Microsoft.AspNetCore.Razor.Test/Compilation/TagHelpers/TagHelperDescriptorTest.cs

```diff
@@ -200,7 +203,47 @@ public void TagHelperDescriptor_WithIndexerAttributes_CanBeSerialized()
                 $"\"{ nameof(TagHelperDescriptor.AllowedChildren) }\":[\"allowed child one\",\"allowed child two\"]," +
                 $"\"{ nameof(TagHelperDescriptor.RequiredParent) }\":\"parent name\"," +
                 $"\"{ nameof(TagHelperDescriptor.TagStructure) }\":0," +
-                $"\"{ nameof(TagHelperDescriptor.DesignTimeDescriptor) }\":null}}";
+                $"\"{ nameof(TagHelperDescriptor.DesignTimeDescriptor) }\":null," +
+                $"\"{ nameof(TagHelperDescriptor.PropertyBag) }\":{{}}}}";
+
+            // Act
+            var serializedDescriptor = JsonConvert.SerializeObject(descriptor);
+
+            // Assert
+            Assert.Equal(expectedSerializedDescriptor, serializedDescriptor, StringComparer.Ordinal);
```

Comment:
You can drop the `StringComparer.Ordinal` parameter. Xunit does ordinal comparisons by default. Similarly below.